roles/neutron-common/meta/main.yml

```diff
@@ -27,3 +27,10 @@ dependencies:
     dependent_packages: "{{ neutron.osp_sys_packages }}"
     rootwrap: True
     when: openstack_install_method == 'distro'
+  - role: openstack-distro
+    project_name: neutron-lbaas
+    dependent_packages: "{{ neutron.osp_sys_packages }}"
+    rootwrap: True
```

Comment:
I think we could just put the neutron lbaas package into the osp_sys_packages variable in the neutron-common defaults: (https://github.com/blueboxgroup/ursula/blob/centos/roles/neutron-common/defaults/main.yml#L12)

We can install the package no matter if its' enabled or not, and then configure neutron accordingly if we have neutron.lbaas == enabled.